great_expectations/data_context/data_context/base_data_context.py

```diff
@@ -2250,7 +2255,16 @@ def save_profiler(
         else:
             key = ConfigurationIdentifier(configuration_key=name)
 
-        self.profiler_store.set(key=key, value=profiler.config)
+        response = self.profiler_store.set(key=key, value=profiler.config)
+        if isinstance(response, GeCloudResourceRef):
+            ge_cloud_id = response.ge_cloud_id
```

Comment:
I don't like this very much but it's the cleanest way to get things working as intended! The hope is to refactor this out later on as we shift more Cloud-specific logic into relevant classes.

Without this, the second test I wrote doesn't work.

Reply:
Does this still work if you just attempt to pull the `id` (`ge_cloud_id`) without the `isinstance()` check?
```python
ge_cloud_id = getattr(response, "ge_cloud_id", None)
```

Reply:
hmm I think we need to refactor the `set` response type across the board to clean this up. I'm wary of `getattr` but I also see the merits of using that here. Good with either!